api/server/sdk/schedule_policy.go

```diff
@@ -0,0 +1,151 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sdk
+
+import (
+	"context"
+
+	"github.com/libopenstorage/openstorage/cluster"
+	"github.com/libopenstorage/openstorage/schedpolicy"
+
+	"github.com/libopenstorage/openstorage/api"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// SchedulePolicyServer is an implementation of the gRPC OpenStorageSchedulePolicy interface
+type SchedulePolicyServer struct {
+	api.OpenStorageSchedulePolicyServer
+	//specHandler spec.SpecHandler
+	cluster cluster.Cluster
+}
+
+// Create method creates schedule policy
+func (s *SchedulePolicyServer) Create(
+	ctx context.Context,
+	req *api.SdkSchedulePolicyCreateRequest,
+) (*api.SdkSchedulePolicyCreateResponse, error) {
+
+	if len(req.GetSchedulePolicy().GetName()) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Must supply Schedule name")
+	}
+
+	if len(req.GetSchedulePolicy().GetSchedule()) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Must supply Schedule")
+	}
+
+	err := s.cluster.SchedPolicyCreate(req.GetSchedulePolicy().GetName(), req.GetSchedulePolicy().GetSchedule())
+
+	if err != nil {
+		return nil, status.Errorf(
+			codes.Internal,
+			"failed to create schedule policy: %v",
+			err.Error())
+	}
+
+	return &api.SdkSchedulePolicyCreateResponse{}, err
+}
+
+// Update method updates schedule policy
+func (s *SchedulePolicyServer) Update(
+	ctx context.Context,
+	req *api.SdkSchedulePolicyUpdateRequest,
+) (*api.SdkSchedulePolicyUpdateResponse, error) {
+
+	if len(req.GetSchedulePolicy().GetName()) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Must supply Schedule name")
+	}
+
+	if len(req.GetSchedulePolicy().GetSchedule()) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Must supply Schedule")
+	}
+
+	err := s.cluster.SchedPolicyUpdate(req.GetSchedulePolicy().GetName(), req.GetSchedulePolicy().GetSchedule())
+
+	if err != nil {
+		return nil, status.Errorf(
+			codes.Internal,
+			"failed to update schedule policy: %v",
+			err.Error())
+	}
+
+	return &api.SdkSchedulePolicyUpdateResponse{}, err
+}
+
+// Delete method deletes schedule policy
+func (s *SchedulePolicyServer) Delete(
+	ctx context.Context,
+	req *api.SdkSchedulePolicyDeleteRequest,
+) (*api.SdkSchedulePolicyDeleteResponse, error) {
+
+	if len(req.GetName()) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Must supply Schedule name")
+	}
+
+	err := s.cluster.SchedPolicyDelete(req.GetName())
+
+	if err != nil {
+		return nil, status.Errorf(
+			codes.Internal,
+			"failed to delete schedule policy: %v",
+			err.Error())
+	}
+
+	return &api.SdkSchedulePolicyDeleteResponse{}, err
+}
+
+// Enumerate method enumerates schedule policy
+func (s *SchedulePolicyServer) Enumerate(
```

Comment:
This API does not make sense to me. The Golang interface 

https://github.com/libopenstorage/openstorage/blob/055b88ca745e1bf046f0cb8d8ba96899f3bf66e8/schedpolicy/sched_policy.go#L17-L18

How can it get `the ones specified` if it takes no arguments?

I think that is incorrect. Enumerate should just return them all since there are no "flags" to match. No need to take `Name` as an argument.

What do you guys think @ipochi @ram-infrac ?